bundles/org.openhab.binding.ambientweather/src/main/java/org/openhab/binding/ambientweather/internal/processor/Ws1400ipProcessor.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ambientweather.internal.processor;
+
+import static org.openhab.binding.ambientweather.internal.AmbientWeatherBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.library.unit.ImperialUnits;
+import org.eclipse.smarthome.core.library.unit.SmartHomeUnits;
+import org.openhab.binding.ambientweather.internal.handler.AmbientWeatherStationHandler;
+import org.openhab.binding.ambientweather.internal.model.EventDataJson;
+import org.openhab.binding.ambientweather.internal.util.PressureTrend;
+
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * The {@link Ws1400ipProcessor} is responsible for updating
+ * the channels associated with the WS-1400-IP series weather stations in
+ * response to the receipt of a weather data update from the Ambient
+ * Weather real-time API.
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+@NonNullByDefault
+public class Ws1400ipProcessor extends AbstractProcessor {
+
+    // Used to calculate barometric pressure trend
+    private PressureTrend pressureTrend = new PressureTrend();
+
+    public Ws1400ipProcessor() {
+        // Set the number of remote sensor channels supported by this station
+        remoteSensor.setNumberOfSensors(8);
+    }
+
+    @Override
+    public void processInfoUpdate(AmbientWeatherStationHandler handler, String station, String name, String location) {
+        // Update name and location channels
+        logger.debug("Station {}: Updating station information channels", station);
```

Comment:
Don't log infront of framework update calls. They are already logged

Reply:
Done for all processors.